pipelines/_ingredients/multiprofit_fit.yaml

```diff
@@ -0,0 +1,62 @@
+description: |
+  Fit a double Gaussian PSF model and various object models with MultiProFit
+parameters:
+  add_pointsource: False
+  fix_centroid: False
+  use_shapelet_psf: False
+tasks:
+  fit_psf_multiprofit:
+    class: lsst.meas.extensions.multiprofit.pipetasks_fit.MultiProFitCoaddPsfFitTask
+    config:
+      python: |
+        from lsst.meas.extensions.multiprofit.pipetasks_fit import MultiProFitCoaddPsfFitTask
+  fit_src_gauss_multiprofit:
```

Comment:
First things first... I'm not opposed to dropping the `_multiprofit` suffix - for the remaining comments, I'll drop it for brevity. It depends on whether you think task labels should include the algorithm or not. Since these are based on generic pipetasks, in principle the subtask could be swapped out to a different algorithm/code. Once one actually gets to the stage of running multiple algorithms concurrently then they actually need different names. I assume this is going to happen with cell vs direct coadds shortly.

Reply:
For the PSF fit, it does not make sense to include `deblend(ed)` in the name. It should probably be called `fit_deepCoadd_psf`, in anticipation of there being a `fit_cellCoadd_psf` in the near future.

Reply:
I'm not so much defining a PSF fit as a double Gaussian as pointing out that it is a current limitation of the code, which might one day be lifted. I'm not so fond of the natural language scheme once it gets to something like `QuadrupleGaussian` or `QuintupleGaussian`. I think at the point we'll have wished the scheme was `_gaussian_4` or something (this might need some #dm-naming-things workshopping). Also this isn't just for tasks labels but likely for plot labels and metric names.

Reply:
`ser`->`sersic` I'm okay with but I'm unhappy with `exponential` being so many characters, while not really having a good abbreviation longer than `exp` (`exponen`, `exponent` are unsatisfactory, I think). We've both seemingly acknowledged that `DeVaucouleurs` is going to be too long and difficult to spell for some users (with all due respect to Gérard Henri). It's true that for existing PSF models we have `DoubleShapeletPsf`, but presumably SDSS went with `CModel` rather than `ExponentialDeVaucouleursLinearMixture` for a reason. So I'm reluctant to adopt a naming scheme that could end up with something as long as `PointsourceExponentialDeVaucouleurs`, hence the `psexpdev` abbreviation. Maybe it should be `ps1_exp1_dev1` for clarity, as above, even if we never intend to fit more than one component of a given type except for `sersic`. But if the composite models warrant a short form `expdev` then I'm not sure making the single-component versions a longer form is actually helpful.

Granted, whatever bulge-disk fit we end up doing will get transformed into the existing object table columns with `Bulge` and `Disk` column names (unless we change that schema), but I still would like to keep internal multiprofit nomenclature consistent.